scripts/zephyr_module.py

```diff
@@ -136,12 +141,12 @@ def process_cmake(module, meta):
     cmake_file = os.path.join(cmake_path, 'CMakeLists.txt')
     if os.path.isfile(cmake_file):
         return('\"{}\":\"{}\":\"{}\"\n'
-               .format(module_path.name,
+               .format(meta.get('name'),
```

Comment:
Why isn't this `meta['name']`? And below also.

Reply:
Probably because I had just been writing `meta.get('name', module_path.name)` 5 seconds before.

Fixed.